cli/vm/cli_test.go

```diff
@@ -93,12 +94,14 @@ func newTestVMCLIWithLogoAndCustomConfig(t *testing.T, printLogo bool, cfg *conf
 	}
 	var c config.Config
 	if cfg == nil {
-		configPath := "../../config/protocol.unit_testnet.single.yml"
+		configPath := filepath.Join("..", "..", "config", "protocol.unit_testnet.single.yml")
 		var err error
-		c, err = config.LoadFile(configPath, "../../config")
+		c, err = config.LoadFile(configPath, filepath.Join("..", "..", "config"))
 		require.NoError(t, err, "could not load chain config")
-		require.Equal(t, "../../testdata/wallet1_solo.json", c.ApplicationConfiguration.Consensus.UnlockWallet.Path)
-		require.Equal(t, "/notary_wallet.json", c.ApplicationConfiguration.P2PNotary.UnlockWallet.Path)
+		require.Equal(t, filepath.Join("..", "..", "testdata", "wallet1_solo.json"), c.ApplicationConfiguration.Consensus.UnlockWallet.Path)
+		if runtime.GOOS != "windows" {
+			require.Equal(t, "/notary_wallet.json", c.ApplicationConfiguration.P2PNotary.UnlockWallet.Path)
+		}
```

Comment:
Remove the notary-related section from this config, because we don't have notary enabled in unit testchains. We don't need a separate config for Windows for these VM tests, existing config should be sufficient, check this.